packages/react-components/index.js

```diff
@@ -10,7 +10,6 @@ module.exports = (neutrino, opts = {}) => {
     html: process.env.NODE_ENV === 'development' && {
       title: 'React Preview'
     },
-    manifest: process.env.NODE_ENV === 'development',
     externals: opts.externals !== false && {},
```

Comment:
@edmorley any idea what purpose this was serving, if any?

Reply:
I have no idea :-/

@eliperelman / @helfi92 do you?